fala/common/footer_views.py

```diff
@@ -1,17 +1,26 @@
 # coding=utf-8
 from fala.common.mixin_for_views import CommonContextMixin, TranslationMixin
 from django.views.generic import TemplateView
+from django.utils.http import url_has_allowed_host_and_scheme
 
 
 class FooterView:
     def get(self, request, **kwargs):
         context = self.get_context_data(**kwargs)
+
+        referer = request.META.get("HTTP_REFERER", "/")
```

Comment:
@willc-work I understand the rationale to add the `previous_url` to the `FooterView` so that it is across all the pages on the app. 

But as the `previous_url` is only used on the `CookiesView`, show we not give the context to that view only?

Reply:
Yes good catch, I initially thought I needed it on every page to be able to capture the referring page and then forgot to revisit that assumption when it worked.